src/bootstrap/mk/Makefile.in

```diff
@@ -72,20 +73,13 @@ check-aux:
 	$(Q)MIRIFLAGS="-Zmiri-disable-isolation" \
 		$(BOOTSTRAP) miri --stage 2 library/std \
 		$(BOOTSTRAP_ARGS) \
-		--no-doc -- \
-		--skip fs:: --skip net:: --skip process:: --skip sys::fd:: --skip sys::pal::
-	$(Q)MIRIFLAGS="-Zmiri-disable-isolation" \
-		$(BOOTSTRAP) miri --stage 2 library/std \
-		$(BOOTSTRAP_ARGS) \
-		--doc -- \
 		--skip fs:: --skip net:: --skip process:: --skip sys::fd:: --skip sys::pal::
```

Comment:
I don't know why I have split up this call into separate `--no-doc` and `--doc` invocations... we'll see if anything breaks I guess.^^

Reply:
I think I may have done it to get separate timings. It doesn't seem worth keeping that.